lib/tasks/deployment/20190822095112_add_home_nation_calendars.rake

```diff
@@ -0,0 +1,26 @@
+namespace :after_party do
+  desc 'Deployment task: add_home_nation_calendars'
+  task add_home_nation_calendars: :environment do
+    puts "Running deploy task 'add_home_nation_calendars'"
+
+    england_and_wales_calendar_area = CalendarArea.find_by(title: 'England and Wales')
+    scotland_calendar_area = CalendarArea.find_by(title: 'Scotland')
+
+    # Put your task implementation HERE.
+    eawca = Calendar.where(calendar_area: england_and_wales_calendar_area, template: true, title: england_and_wales_calendar_area.title).first_or_create!
+    sca = Calendar.where(calendar_area: scotland_calendar_area, template: true, title: scotland_calendar_area.title).first_or_create!
+
+    Calendar.find_by(title: 'Bath and North East Somerset (BANES)').update(based_on_id: eawca.id)
+    Calendar.find_by(title: 'Frome').update(based_on_id: eawca.id)
+    Calendar.find_by(title: 'Sheffield').update(based_on_id: eawca.id)
+    Calendar.find_by(title: 'Oxfordshire').update(based_on_id: eawca.id)
+
+    highland_calendar = Calendar.find_by(title: 'Highland')
+    highland_calendar.update(based_on_id: sca.id) if highland_calendar
```

Comment:
Do we need to re-populate the Highland template calendar with the scotland bank holidays? I think it will have the England and Wales ones now.

Reply:
There's a job to sort this now